### DISCUSSION TITLE
Welcome to lvlmindbeta Discussions!
You can ask us questions about our repository.

### DISCUSSION THREAD

**Discussion by josoavj** [josoavj/lvlmindapp#1] [Announcements]
Hello, I am Josoa one of the contributors to the lvlmindapp repository.  Currently I am working on mobile development with the flutter framework (Dart) especially in this project.  I am open to all kinds of comments or even suggestions and recommendations from you on the improvement of my work and also my projects.
@josoavj

**Comment by koulienathalie** [comment]
Can you add me as a contributor

**Reply by josoavj** [reply]
![b16323c1c4fcbc6b9e1f790f931af00d.jpg](https://github.com/Josoa886/lvlmindapp/assets/76913187/11bde99b-dfec-47e7-a796-02e77565b2e6)

**Comment by josoavj** [comment]
@foulburst 
@koulienathalie

**Reply by foulburst** [reply]
👋
